### PR TITLE
Fixing query string for sync and adding readme

### PR DESCRIPTION
**Pull request by ishidas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Endpoint  | Implementation
 
 The group service behind the GWS API uses LDAP caches to improve reliability and response time for some requests. The caches may take several seconds to update after changes to GWS are made. This can cause inconsistency in that you may not 'read what you wrote' during the time the cache is invalid.
 
-If you need to ensure cache consistency, you can add `sync: true` to the options of group create and membership add operations.
+If you need to ensure cache consistency, you can add `sync: true` to the options of group create and membership add operations. Note: query sting GWS is expecting is `/?synchronize` not `/?synchronize=true`
```
Comment (ccloops) [inline README.md]: should this be synchronized instead of synchronize ?

Comment (ishidas) [reply README.md]: ah good catch 🎣

Comment (kevintechie) [reply README.md]: The note should go in the code, not here.

// Note: For synchronized PUT calls, GWS expects the /?synchronzed query parameter without a value (i.e.: not /?synchronized=ture).

File: README.md [hunk 2]
```diff
@@ -114,4 +114,8 @@ If you need to ensure cache consistency, you can add `sync: true` to the options
     # put appropriate values in the new config.js
     npm test
 
-It is recommended that you use the eval environment (https://eval.groups.uw.edu/group_sws/v3/) when developing/testing to avoid making lasting changes. And the search test will occasionally timeout if you are using prod.
\ No newline at end of file
+It is recommended that you use the eval environment (https://eval.groups.uw.edu/group_sws/v3/) when developing/testing to avoid making lasting changes. And the search test will occasionally timeout if you are using prod.
+
+## Caveat on using replaceMembership method
```
Comment (kevintechie) [inline README.md]: This warning should go in the implementation table column for replaceMembership or directly below the table of methods .

**Pull request merged**